graylog2-web-interface/src/logic/authentication/okta/types.ts

```diff
@@ -20,22 +20,25 @@ import * as Immutable from 'immutable';
 import AuthenticationBackend from 'logic/authentication/AuthenticationBackend';
 
 export interface OktaBackendConfig {
-  type: 'okta';
-  oktaBaseUrl: string;
+  type: string;
+  baseUrl?: string;
+  oktaBaseUrl?: string;
```

Comment:
Should we change this types like we did in enterprise? That is, to have one for OIDC and one for Okta.